moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -335,7 +338,7 @@ void ServoCalcs::run(const ros::TimerEvent& timer_event)
     zero_velocity_count_ = 0;
   }
 
-  if (ok_to_publish_)
+  if (ok_to_publish_ && !stop_requested_)
   {
     // Put the outgoing msg in the right format
```

Comment:
Or is it that sometimes we'll publish one more message than we should be after stop is requested because it happened while we were executing this?

Reply:
yep, that was also an issue